src/css/override_vars.scss

```diff
@@ -23,29 +23,89 @@ $color-secondary-lightest:   #deeeef; // DEEF!
 $color-focus:                rgba(255, 221, 0, 0.55);
 $color-visited:              #4c2c92;
 
-$color-gold:                 #f8ba1c;
+$color-gold:                 #f4b400;
 
-$color-finish:               #0daf02;
+$color-finish:               #0daf00;
 $color-warning:              #cb4f03;
-$color-error:                #f16031;
+$color-error:                #cb4f03;
+$color-red:                #cb4f03;
 
 $color-gray:                 #595959;
-$color-gray-dark:            darken($color-gray, 20%);
+$color-darkgray: #555;
+$color-mediumgray: #8e8e8e;
+$color-lightgray: #d3d3d3;
+$color-textblack: #333;
+$color-trueblack: #000;
+
+$color-gray-dark: #555; // redundant to prevent breaking
 
 $color-disclaimer-text:      #000000;
 $color-disclaimer-link:      #0071bc;
 
-$color-base:                 $color-gray;
-$text-max-width:             660px;
+$color-base:                 $color-textblack;
+$text-max-width:             36em;
 
 // TODO what color is this? it should follow convention
-$color-lightgray:            #bababa;
 $site-max-width:             1040px;
 
 $color-background-light:     #f3f4f8;
+$color-cream:     #f5f5f3;
 
 $nav-width:                  600px;
 
 $font-sans:                  '18Franklin', system, -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', 'Helvetica', 'Arial', sans-serif;
 $font-serif:                 'Georgia', 'Cambria', 'Times New Roman', 'Times', serif;
 $font-mono:                  'Roboto Mono', 'Bitstream Vera Sans Mono', 'Consolas', 'Courier', monospace;
+
+$base: 8;
+$base-grid: #{$base}px;
+$base-rem: #{$base}rem;
+$base-type: $base * 2;
+$base-type-rem: #{$base * 2}rem;
+$base-type-px: #{$base * 2}px;
+
+$bx5: #{($base / 2 / $base-type)}rem;    // 4px
```

Comment:
I'm not sure I like the name of these, mainly because I don't know what they mean. Can you rename to something more understandable?